completions/ip

```diff
@@ -1,6 +1,6 @@
 # ip(8) completion                                         -*- shell-script -*-
 
-_iproute2_etc()
+_comp_cmd__iproute2_etc()
```

Comment:
Kind of like https://github.com/scop/bash-completion/pull/883#discussion_r1113464662 this, too

Reply:
Changed to `_comp_cmd_ip__iproute2_etc` in e0ceb19a891055ab2869aac7be2c800e0cd04c18 per https://github.com/scop/bash-completion/pull/883#discussion_r1174530020